python/mxnet/rnn/rnn_cell.py

```diff
@@ -284,10 +304,14 @@ def unroll(self, length, inputs, begin_state=None, layout='NTC', merge_outputs=N
 
         Returns
         -------
-        outputs : list of Symbol
-            output symbols.
+        outputs : list of Symbol or Symbol
+            Symbol (if `merge_outputs` is True) or list of Symbols
```

Comment:
we should describe the shape of output & state, probably in a separate section like in pytorch
like
(batch_size, length, *) (layout='NTC') or [(batch_size, *)]*length